section3/exercises/arrays.rb

```diff
@@ -17,51 +17,59 @@
 # EXAMPLE: write code below that will print an array of animals.
 # Store the array in a variable.
 animals = ["Zebra", "Giraffe", "Elephant"]
-print animals
+p animals
 
 # EXAMPLE: Write code below that will print "Zebra" from the animals array
 # YOUR CODE HERE
-print animals[0]
+p animals[0]
 
 # YOU DO: Write code below that will print the number of elements in array of
 # animals from above.
-
+p animals.length
 
 # YOU DO: Write code that will reassign the last item in the animals
 # array to "Gorilla"
-
-
+p animals[2] = "Gorilla"
+p animals
 # YOU DO: Write code that will add a new animal (type of your choice) to position 3.
-
+animals.push("Pigeon")
+p animals
 
 # YOU DO: Write code that will print the String "Elephant" in the animals array
-
+animals[2] ="Elephant"
+p animals[2]
 
 #-------------------
 # PART 2: Foods: Array Methods
 #-------------------
 
 # YOU DO: Declare a variable that will store an an array of at least 4 foods (strings)
-
+foods = ["bologna", "falafel", "bagel", "garbanzo beans"]
 
 # YOU DO: Write code below that will print the number of elements in the array of
 # foods from above.
-
+p foods.length
 
 # YOU DO: Write code below that uses a method to add "broccoli" to the foods array and
 # print the changed array to verify "broccoli" has been added
-
+foods.push("broccoli")
+p foods
 
 # YOU DO: Write code below that removes the last item of food from the foods array and
 # print the changed array to verify that item has been removed
+foods.pop()
```

Comment:
In Ruby, we generally won't use empty parens `()` on a method call without any arguments. So we can just say `foods.pop`. Technically works either way though!